src/axom/core/Array.hpp

```diff
@@ -199,24 +196,20 @@ class Array : public ArrayBase<T, DIM, Array<T, DIM, SPACE>>
    * \param [in] allocator_id the ID of the allocator to use (optional)
```

Comment:
```suggestion
   * \param [in] allocator_id the ID of the allocator to use (optional).
   *     Default value of -1 causes the new Array to use the allocator
   *     of \a other.
```
or words to that effect.